src/ui/guides/02_is_my_vpn_working_correctly.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "is_my_vpn_working_correctly",
+  "title": "Is my VPN connected to the right server location?",
+  "image": "qrc:/guides/02_is_my_vpn_working_correctly.svg",
+  "blocks": [
+    { "id": "c_1",
+      "type": "text",
+      "content": "After signing on to Mozilla VPN and connecting to one of our 400+ servers in over 30 different countries, you can check if your VPN is active in your chosen geolocation in two easy ways:" },
+    { "id": "c_2",
+      "type": "title",
+      "content": "Search Google" },
+    { "id": "c_3",
+      "type": "text",
+      "content": "Head to Google’s homepage and you will immediately see your location status (as well as any change in language) on display." },
```

Comment:
Not a localization issue, but this is off. 

Language on Google is influenced by several factors, location being only one of them. If you're logged in with your account, the language won't change and will follow your account's settings.

Reply:
Plus I don't like pushing for google in the VPN product. I'll discuss this with the UX team.